README.md

```diff
@@ -46,59 +59,46 @@ of existing clusters, rather than deploying new one.
 
 - Automated discovery of SAP HANA HA clusters;
 - SAP Systems and Instances overview;
-- Configuration validation for Pacemaker, Corosync, SBD, SAPHanaSR and other generic _SUSE Linux Enterprise for SAP Application_ OS settings (a.k.a. the _HA Checks_);
+- Configuration validation for Pacemaker, Corosync, SBD, SAPHanaSR and other generic _SUSE Linux Enterprise for SAP Application_ OS settings (a.k.a. the _HA Config Checks_);
 - Specific configuration audits for SAP HANA Scale-Up Performance-Optimized scenarios deployed on MS Azure cloud.
 
-# Base concepts
+# Introduction
 
-The entire Trento application is composed of the following parts:
+_Trento_ is a comprehensive monitoring solution made by two main components, the _Trento Server_ and the _Trento Agent_.
 
-- One or more Consul Agents in server mode;
-- The Trento Web UI (`trento web`);
-- A Consul Agent in client mode for each target node;
-- A Trento Agent (`trento agent`) for each target node.
+The _Trento Server_ is an independent, cloud-native, distributed system and should run on dedicated infrastructure resources. It is in turn composed by the following sub-systems:
 
-> See the [architecture document](./docs/trento-architecture.md) for additional details.
+- The `trento web` application;
+- The `trento runner` worker;
+- A [Consul] data-plane;
+- An [ARA] service.
 
-# Requirements
+The _Trento Agent_ is a single background processes (`trento agent`) running in each host of the target infrastructure the user desires to monitor.
 
-While the `trento web` component has only been tested on openSUSE 15.2
-and SLES 15SP2 so far, it should be able to run on most modern Linux distributions.
+See the [architecture document](./docs/trento-architecture.md) for additional details.
 
-The `trento agent` component could in theory also run on openSUSE, but it does not make much sense as it
-needs to interact with different low-level SAP applications components
-which are expected to be run in a
-[SLES for SAP](https://www.suse.com/products/sles-for-sap/) installation.
+> Being the project in development, all of the above might be subject to change!
 
-## Runtime dependencies
+# Installation
 
-Running the application will require:
+## Requirements
 
-- A running [Consul](https://www.consul.io/downloads) cluster.
+The _Trento Server_ is intended to run in many ways, depending on users' already existing infrastructure, but it's designed to be cloud-native and OS agnostic.
```

Comment:
A leftover `'` in `users'`

Reply:
Nope, that's actually plural saxon genitive, i.e. "infrastructure of users" 🙂